book/src/secrets-and-credentials.md

```diff
@@ -48,6 +48,13 @@ The value of both parameters may be a literal or a template containing the follo
 
     * `${pv.name}`
       * Automatically replaced with the name of the `PersistentVolume` object being provisioned at provision.
+    * `${pvc.namespace}`
+      * Automatically replaced with the namespace of the `PersistentVolumeClaim` object being provisione
+
+The value of `csi.storage.k8s.io/provisioner-secret-name` also supports the following template variables which are automatically replaced by the `external-provisioner` at provision time:
+
+  * `${pvc.name}`
+    * Automatically replaced with the name of the `PersistentVolumeClaim` object being provisioned.
 
 If specified, the CSI `external-provisioner` will attempt to fetch the secret before provisioning and deletion.
```

Comment:
Can you also add a note here that for deletion, if the entire namespace was deleted (including the secret needed for deletion), then no secret will be passed to the delete call.  If the driver requires a deletion secret and this happens, then the volume and the PV may need to be manually cleaned up.

Reply:
Added a note for this :+1: